numpy/lib/index_tricks.py

```diff
@@ -45,8 +45,11 @@ def ix_(*args):
     Returns
     -------
     out : tuple of ndarrays
-        N arrays with N dimensions each, with N the number of input
-        sequences. Together these arrays form an open mesh.
+        N arrays (of integer or boolean type) with N dimensions each, with N
```

Comment:
Before I amend the correction, should I append `[skip ci]`? 

The tests obviously pass (or don't even start).